src/App.tsx

```diff
@@ -1,42 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import { Tabs } from './components/Tabs';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-export const tabs = [
+import { Tab } from './types/Tab';
+
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(1);
+
+  const onTabSelected = (tabId: number): void => {
```

Comment:
It's extra, TS already know about that
```suggestion
  const onTabSelected = (tabId: number) => {
```